### PR TITLE
[11.0] [MIG] purchase_request_department

### PR DESCRIPTION
**Pull request by HviorForgeFlow**
Migration to 11. Minor changes.

### CODE REVIEW

File: oca_dependencies.txt [hunk 1]
```diff
@@ -1 +1,3 @@
 server-tools
+hr
+purchase-workflow https://github.com/Eficent/website.git 11.0-mig-purchase_request
```
Comment (JordiBForgeFlow) [inline oca_dependencies.txt]: This link does not seem to be valud. website?

File: purchase_request_department/__init__.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
```
Comment (nikul-serpentcs) [inline purchase_request_department/__init__.py]: @hveficent Remove Copyright from all `__init__` file

File: purchase_request_department/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl-3.0).
+
+{
+    "name": "Purchase Request Department",
+    "author": "Eficent, "
```
Comment (nikul-serpentcs) [inline purchase_request_department/__manifest__.py:6]: IMO Add `website` key also as per [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L11)

File: purchase_request_department/__manifest__.py [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    "category": "Purchase Management",
+    "post_init_hook": "post_init_hook",
+    "depends": [
+        "hr",
+        "purchase_request"
+    ],
+    "data": [
+        "views/purchase_request_department_view.xml"
+    ],
+    'demo': [],
```
Comment (nikul-serpentcs) [inline purchase_request_department/__manifest__.py]: If not use then remove it

File: purchase_request_department/hooks.py [hunk 5]
```diff
@@ -0,0 +1,18 @@
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl-3.0).
+from openerp import api, SUPERUSER_ID
```
Comment (nikul-serpentcs) [inline purchase_request_department/hooks.py]: odoo

**Review by nikul-serpentcs — COMMENTED** [review]
Improve Code

**Comment by HviorForgeFlow** [comment]
All comments attended @nikul-serpentcs @jbeficent .
Thanks for reviewing :smiley:

File: purchase_request_department/README.rst [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+
+.. image:: https://odoo-community.org/logo.png
+   :alt: Odoo Community Association
+   :target: https://odoo-community.org
+
+OCA, or the Odoo Community Association, is a nonprofit organization whose
+mission is to support the collaborative development of Odoo features and
+promote its widespread use.
+
+This module is part of the `OCA/purchase-workflow <https://github.com/OCA/purchase-workflow/tree/11.0/purchase_request_department>`_ project on GitHub.
```
Comment (JayVora-SerpentCS) [inline purchase_request_department/README.rst:88]: is it necessary to write?

File: purchase_request_department/models/purchase_request.py [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+from odoo import api, fields, models
+
+
+class PurchaseRequest(models.Model):
+    _inherit = 'purchase.request'
+
+    def _get_my_department(self):
+        employees = self.env.user.employee_ids
+        return (employees[0].department_id if employees
+                else self.env['hr.department'])
```
Comment (JayVora-SerpentCS) [inline purchase_request_department/models/purchase_request.py]: does not this return env object and not Id False for the default to digest ?

**Review by JayVora-SerpentCS — CHANGES_REQUESTED** [review]

File: purchase_request_department/README.rst (hunk 6 above)
Comment (HviorForgeFlow) [reply purchase_request_department/README.rst:88]: It is autogenerated

File: purchase_request_department/models/purchase_request.py (hunk 7 above)
Comment (HviorForgeFlow) [reply purchase_request_department/models/purchase_request.py]: What should it be done to solve this?? @JayVora-SerpentCS

Comment (JayVora-SerpentCS) [reply purchase_request_department/models/purchase_request.py]: IMHO, return (employees[0].department_id if employees else False) should do. I could be wrong, but you may check.

File: oca_dependencies.txt [hunk 8]
```diff
@@ -1,3 +1,5 @@
 server-tools
 product-attribute
 server-ux
+hr
+purchase-workflow https://github.com/Eficent/purchase-workflow.git 11.0-mig-purchase_request
```
Comment (LoisRForgeFlow) [inline oca_dependencies.txt]: dependency merged, can you remove this and rebase?

Comment (HviorForgeFlow) [reply oca_dependencies.txt]: done!

**Review by LoisRForgeFlow — APPROVED** [review]
Thanks :+1:

**Comment by HviorForgeFlow** [comment]
Update review
CC: @jbeficent @JayVora-SerpentCS @nikul-serpentcs

**Review by nikul-serpentcs — APPROVED** [review]
Minor change
`self.env.ref('purchase_request.group_purchase_request_user')` use as globel variable, 
otherwise LGTM :+1:

**Pull request merged**